.github/workflows/destroy-deployment.yml

```diff
@@ -0,0 +1,13 @@
+name: Destroy-deployment
+
+on:
+  pull_request:
+    types:
+      - closed
+jobs:
+  destroy-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
```

Comment:
What would happen if one opens a link to the destroyed deployment?

Reply:
The page will be redirected to the last version